vendor/assets/schemas/oai-pmh/successful_response_combined.xsd.erb

```diff
@@ -0,0 +1,33 @@
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema">
+
+  <xs:annotation>
+    <xs:documentation>
+      The xml schemas need to be combined like this because Nokigiri uses libxml which has the limitation that all
+      schema declarations must be in one file.
+      https://mail.gnome.org/archives/xml/2009-November/msg00022.html
+      Additionally, Nokigiri does not download remote schemas so we downloaded them and store them locally.
+
+      This XSD is used for testing of the OaiController.
+    </xs:documentation>
+  </xs:annotation>
+
+  <xs:import
+    namespace="http://www.openarchives.org/OAI/2.0/"
+    schemaLocation="<%= Rails.root %>/vendor/assets/schemas/oai-pmh/sub-schemas/successful_response.xsd"
```

Comment:
I went with just the relative paths the way it is in the IEEE LOM files to avoid the template engine. I noticed we also did not need Rails.root.join for the schema paths when reading the schema files. So in oai_controller.rb we can use this:
```SUCCESSFUL_RESPONSE_XSD = 'vendor/assets/schemas/oai-pmh/successful_response_combined.xsd'```
instead of this:
```SUCCESSFUL_RESPONSE_XSD = Rails.root.join('vendor/assets/schemas/oai-pmh/successful_response_combined.xsd.erb')```

I changed this in lom/tasks_controller.rb too for consistency.

Reply:
Awesome, that's a good thing! :+1: